qfieldcloud_sdk/sdk.py

```diff
@@ -231,7 +392,21 @@ def upload_files(
         force: bool = False,
         job_id: str = "",
     ) -> List[Dict]:
-        """Upload files to a QFieldCloud project"""
+        """Upload files to a QFieldCloud project.
+
+        Args:
+            project_id (str): The ID of the project to upload files to.
```

Comment:
```suggestion
            project_id (str): Project ID.
```

And so on, it's not needed to be very verbose about this.